third_party/bitcoin-core/BUILD.gn

```diff
@@ -0,0 +1,54 @@
+config("warnings") {
+  cflags = []
+
+  if (is_clang) {
+    cflags += [ "-Wno-unused-function" ]
+  }
+}
+
+config("bitcoin-core_config") {
+  include_dirs = [ "src/src" ]
+}
+
+static_library("bitcoin-core") {
+  configs -= [ "//build/config/compiler:chromium_code" ]
+  configs += [
+    "//build/config/compiler:no_chromium_code",
+    "//build/config/compiler:exceptions",
+    ":warnings",
+  ]
+
+  public_configs = [ ":bitcoin-core_config" ]
+
+  sources = [
```

Comment:
please add all indirectly used headers like uint256.h

Reply:
also please add include rules to exclude third_party/bitcoin-core globally and then add allow rules in brave_wallet for the specific headers it needs to directly include

Reply:
and please add a comment here that any files that are added need security review and probably add someone as code owner for this file

Reply:
fixed in 83746af845762b01b740e985ad0ec7a10ac90a93